css/cssom/cssstyledeclaration-csstext.html

```diff
@@ -97,6 +97,10 @@
 
         }, 'invalid property does not appear');
 
+        // https://github.com/w3c/csswg-drafts/issues/1033
+        test(function() {
+            assert_equals(getComputedStyle(document.body).cssText, "");
```

Comment:
I suggest having a new element with several properties set, and then check its computed `cssText`.

Reply:
Yeah, good call, done.